invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/ShareOptions/ShareModal.js

```diff
@@ -10,7 +10,7 @@ import { Icon, Button, Modal, Tab } from "semantic-ui-react";
 import { i18next } from "@translations/invenio_app_rdm/i18next";
 import PropTypes from "prop-types";
 import { LinksTab } from "./AccessLinks/LinksTab";
-// import { GrantsTab } from "./GrantsTab";
+import { AccessRequestsTab } from "./AccessRequests/AccessRequestsTab";
 
 export const ShareModal = ({ record, open, handleClose, accessLinksSearchConfig }) => {
```

Comment:
you will need to make it a class component. We need to preserve the state of the record somewhere and we will do it here:
```suggestion
export class ShareModal extends Component { 

constructor(props){
this.state = {record: props.record} // I leave the nice formatting and deconstructing assignment to you
}

```
// your callback function to use when change settings endpoint returns updated record
handleRecordUpdate = (updatedRecord) => {
this.setState = {record: updatedRecord}'
}